demos/python_demos/cocosnet_demo/cocosnet_demo.py

```diff
@@ -0,0 +1,142 @@
+from preprocessing import preprocess_with_images, preprocess_with_semantics, preprocess_for_seg_model
+from postprocessing import postprocess, save_result
+
+
+def build_argparser():
+    parser = ArgumentParser(add_help=False)
+    args = parser.add_argument_group('Options')
+    args.add_argument('-h', '--help', action='help', default=SUPPRESS,
+                      help='Show this help message and exit.')
+    args.add_argument("-c", "--correspondence_model",
+                      help="Required. Path to an .xml file with a trained correspondence model",
+                      required=True, type=str)
+    args.add_argument("-g", "--generative_model",
+                      help="Required. Path to an .xml file with a trained generative model",
+                      required=True, type=str)
+    args.add_argument("-s", "--segmentation_model",
+                      help="Required. Path to an .xml file with a trained semantic segmentation model",
+                      required=False, default=None, type=str)
+    args.add_argument("-ii", "--input_images",
+                      help="Required. Path to a folder with input images or path to a input image",
+                      default="", type=str)
+    args.add_argument("-is", "--input_semantics",
+                      help="Required. Path to a folder with semantic images or path to a semantic image",
+                      default="", type=str)
+    args.add_argument("-ri", "--reference_images",
+                      help="Required. Path to a folder with reference images or path to a reference image",
+                      default="", type=str)
+    args.add_argument("-rs", "--reference_semantics",
+                      help="Required. Path to a folder with reference semantics or path to a reference semantic",
+                      default="", type=str)
+    args.add_argument("-o", "--output_dir", help="Path to directory to save the results",
+                      required=False, type=str, default="results")
+    args.add_argument("-l", "--cpu_extension",
+                      help="Optional. Required for CPU custom layers. "
+                           "Absolute MKLDNN (CPU)-targeted custom layers. Absolute path to a shared library with the "
+                           "kernels implementations", type=str, default=None)
+    args.add_argument("-d", "--device",
+                      help="Optional. Specify the target device to infer on; CPU, GPU, FPGA, HDDL or MYRIAD is "
+                           "acceptable. Sample will look for a suitable plugin for device specified. Default value is CPU",
+                      default="CPU", type=str)
+    return parser
+
+
+def get_files(path):
+    if os.path.isdir(path):
+        return os.listdir(path)
+    return [""]
+
+
+def get_mask_from_image(image, model):
+    image = preprocess_for_seg_model(image)
+    res = model.infer(image)
+    mask = np.argmax(res, axis=1)
+    mask = np.squeeze(mask, 0)
+    return mask + 1
```

Comment:
this code work with specific segmentation model only. I think you need to add info about it in readme. Also need to note about which dataset trained segmentation model should be used (e.g. maybe somebody will try to use deeplab v3 model from omz, but it is unsuitable)